docs/pages/virtual.md

```diff
@@ -5,10 +5,21 @@ layout: default
 
 # Virtual environments
 
-| Name                                                     | Short description                                                         | 🚦  |
-| -------------------------------------------------------- | ------------------------------------------------------------------------- | :-: |
-| [Conda](https://docs.conda.io/projects/conda/en/stable/) | Installs, runs, and updates packages and their dependencies.              | 🟢  |
-| [pipenv](https://pipenv.pypa.io/en/latest/)              | Automatically creates and manages a virtualenv for your projects.         | 🟠  |
-| [pyenv](https://github.com/pyenv/pyenv)                  | Lets you easily switch between multiple versions of Python.               | 🟠  |
-| [virtualenv](https://virtualenv.pypa.io/en/latest/)      | Creates isolated Python environments, and offers more features than venv. | 🟠  |
-| [venv](https://docs.python.org/3/library/venv.html)      | Creates isolated Python environments.                                     | 🔴  |
+| Name                    | Short description                                                                                                                                                                         | 🚦  |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: |
+| conda-forge [miniforge] | Installs, runs, and updates packages and their dependencies. A trusted sub-set of `conda` with the `conda-forge` channel.                                                                 | 🟢  |
```

Comment:
```suggestion
| conda-forge [miniforge] | Installs, runs, and updates packages and their dependencies. Similar to `conda`, but uses community maintained packages from `conda-forge` channel instead of commercially maintained packages.                                                                 | 🟢  |
```
Everything on conda-forge is packaged independently to conda, so I think sub-set isn't the right word.

Reply:
Thanks for the comments! That is the specificity I was looking for.

I don;t know where I got "subset" from. I think my brain morphed "minimal installers" into that somehow...

> [...] the minimal installers for [Conda](https://conda.io/) and [Mamba](https://github.com/mamba-org/mamba) specific to [conda-forge](https://conda-forge.org/), [...]

Reply:
It's not really "Similar to" though? It _is_  `conda`, right?

Reply:
How about
> Uses `conda` but with community managed...

Reply:
Ah yes, ☝️ is much better